.github/workflows/require-pr-label.yml

```diff
@@ -4,15 +4,14 @@ on:
   pull_request:
     types: [opened, reopened, labeled, unlabeled, synchronize]
 
-permissions:
-  issues: write
-  pull-requests: write
-
 jobs:
   label-dnm:
     name: DO-NOT-MERGE
     if: github.repository_owner == 'python'
     runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
```

Comment:
These labels are only used on PRs, so I'm pretty sure the `issues: write` can be removed.  In addition, I think we might not even need the `pull-requests: write` since AFAICT we are not writing anything.
I don't have enough energy today to also dig into this to make sure that the permissions are not needed, so feel free to merge this as is and we can open a separate issue to investigate and possibly remove these.

Reply:
Yep, we can check with this separately. This PR isn't adding any extra permissions here.